tests/lib/snaps/config-versions-v2/meta/snap.yaml

```diff
@@ -1,2 +1,7 @@
 name: config-versions
 version: 2.0
+
+apps:
+  sh:
+    command: ../../../bin/sh
```

Comment:
Allright, since you haven't specifically blocked on this and we have quite a few spread tests doing it, I'll treat it as a followup material where we fix all that at once.